database/postgresql/listings-seed.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const csvWriter = require('csv-write-stream');
+const faker = require('faker');
+
+// create stream and name file
+const writeListings = fs.createWriteStream('./database/postgresql/listings1.csv');
+
+// write headers for CSV file
+writeListings.write('id,address,city,state,zip_code,price,realtor_id\n', 'utf8');
+
+const listingsDataGen = (writer, encoding, callback) => {
+  let i = 10000000;
```

Comment:
i think just the same notes as the other seed files, you could also make this code a little drier by condensing the repeated code into a helper function and then passing in the appropriate data fcn